cmd/broadcast/utils.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"bufio"
+	"encoding/hex"
+	ida "github.com/harmony-one/libunison/internal/ida/coopcast"
+	"io"
+	"log"
+	"math/rand"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const pubKeySize int = 20
+
+// Entry is a single config of a node.
+type Entry struct {
+	Sid     string
```

Comment:
Although not required by GoDoc style, I would recommend explaining public members of a `struct`.  Like, untrained eyes would ask things like “What is `Sid`?  Session ID?  Secret ID?  A male name Sid?  Also I see that it's a `string`… What is its syntax?”

Reply:
SimpleID, might be replaced later by more complex ID, I will change it to ID